pkg/codec/opus/params.go

```diff
@@ -16,12 +18,18 @@ type Params struct {
 
 // NewParams returns default opus codec specific parameters.
 func NewParams() (Params, error) {
-	return Params{}, nil
+	return Params{
+		BaseParams: codec.BaseParams{
+			Latency: 20 * time.Millisecond,
```

Comment:
There may be two options:
- Put `Latency time.Duration` to `codec.BaseParams`
  - *pro:* provides a general way to pass expected latency to codecs. Other codecs may also have latency parameter. (e.g. vpx has a option to add some lag to achieve better quality and accurate bit rate control.)
  - *con:* can't disallow values which are not supported by the codec.
- Put `Latency CustomLatencyType` (`type CustomLatencyType time.Duration`) to `opus.Param`
  - *pro:* can restrict value according to the codec spec.
    - however, `CustomLatencyType` can store a value which is not defined as consts. Validation or nearest value select is still needed.
  - *con:* way to set latency will be different for each codec.

What do you think?

Reply:
(I don't have strong opinion about this.)

Reply:
I prefer using `CustomLatencyType`, because this allows users to know available latencies for specific codecs since the options will show up on Godocs. Whereas if we used a generic `time.Duration`, there's really no way to know the right values until the user reads our codebase and understands the specific codec. 

And, yes, I agree that we still have to check the actual value since Go doesn't have a way to enforce it at compile time.

Reply:
Added `Latency` type.